js_modules/dagit/packages/core/src/ui/Filters/useFilter.tsx

```diff
@@ -29,7 +29,7 @@ export const FilterTag = ({
   iconName: IconName;
   onRemove: () => void;
 }) => (
-  <div style={{display: 'inline-block', height: '24px'}}>
+  <div>
```

Comment:
It seems like we can just use a flex row to render tags, wdyt? Wrapping the tag in a `div` should prevent it from stretching weirdly.

Reply:
seems fine to me.